lib/widgets/custom_data_table/custom_data_table.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:efficacy_admin/models/club_position/club_position_model.dart';
+import 'package:flutter/material.dart';
+
+class CustomDataTable extends StatelessWidget{
+  final List<String> columns;
+  final List<ClubPositionModel> rows;
+  final double? columnspace;
+  const CustomDataTable({
+    super.key,
+    this.columnspace,
+    this.columns = const [],
+    this.rows = const [],
+  });
+  
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border.all(color: Colors.grey),
+        borderRadius: BorderRadius.circular(10),
+      ),
+      child: DataTable(
```

Comment:
try adding a bit of padding on the right of the position it is sticking to the edge... You might use relative like by MediaQuery.of(context).size.width*some_factor for the right padding